JS-Algorithms/challenges.js

```diff
@@ -41,7 +60,11 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  if( index < 0 || isNaN(index) ){
+    return null;
+  }
+  index = index % COUNTRY_NAMES.length;
```

Comment:
Here I would avoid manipulating the argument and probably just add that inline

```
COUNTRY_NAMES.slice(index % COUNTRY_NAMES.length)
```

I think as a general rule its best to avoid mutating arguments directly